applications/monitoring/values.yaml

```diff
@@ -24,50 +122,64 @@ chronograf:
       memory: 30Gi
       cpu: 4
 
-  ## Use our own Gafaelfawr ingress.
+  # -- Chronograf ingress
```

Comment:
Needs a `@default` comment to avoid dumping JSON into the README, or documentation for each setting.